tests/plotting/test_plotting.py

```diff
@@ -2618,12 +2491,7 @@ def test_ssao_pass():
         return
 
     pl.enable_ssao()
-    pl.show(auto_close=False)
-
-    # ensure this fails when ssao disabled
-    pl.disable_ssao()
-    with pytest.raises(RuntimeError):
-        pl.show()
+    pl.show()
```

Comment:
Why was this part of the test removed?

Reply:
This test wasn't working as intended, it was failing (not throwing a runtime error) on a clean install on main branch. I believe it works on github actions thanks to an old cache, which causes to throw a runtime error due to images dissimilarity.

Since what this test should raise was confusing for me, I asked @akaszynski and he recommended to remove those lines.